skimage/feature/blob.py

```diff
@@ -25,26 +26,27 @@ def _blob_overlap(blob1, blob2):
     Parameters
     ----------
     blob1 : sequence
-        A sequence of ``(y,x,sigma)``, where ``x,y`` are coordinates of blob
-        and sigma is the standard deviation of the Gaussian kernel which
-        detected the blob.
+        A sequence of ``(r,c,sigma)`` or ``(f,r,c,sigma)``, where ``r,c``
+        (or ``(f,r,c,sigma)``) are coordinates of blob and ``sigma`` is
+        the standard deviation of the Gaussian kernel which detected the blob.
```

Comment:
I would also point out that this needs to be a sequence of arrays, since you do `blob1 - blob2`, which would fail for e.g. tuples. Alternatively, you could add the lines `blob1 = np.asanyarray(blob1)` and `blob2 = np.asanyarray(blob2)`.